.github/workflows/tag.yml

```diff
@@ -18,21 +17,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-        with:
-          submodules: true
-
-      - name: Fetch History
-        run: git fetch --prune --unshallow
-
-      - name: Configure Git User
-        run:  |
-          git config --global credential.helper cache
-          git config remote.origin.url https://${UPBOUND_BOT_USR}:${UPBOUND_BOT_PSW}@$(git config --get remote.origin.url | sed -e 's/https:\/\///')
-          git config user.name "upbound-bot"
-          git config user.email "info@crossplane.io"
 
       - name: Create Tag
-        run: make -j2 tag
-        env:
-          VERSION: ${{ github.event.inputs.version }}
-        
\ No newline at end of file
+        uses: negz/create-tag@v1
```

Comment:
Woo! Does this create annotated tags?

Reply:
https://github.com/negz/create-tag/blob/main/src/main.ts#L25

I believe so.

Reply:
Nice!